### ISSUE TITLE
Checkbox need two click to be activated

### ISSUE THREAD

**Issue by Cele91** [jo3stevens/ACServerManager#20]
As soon as the page loads, all checkboxes are unboxed where to click. At first click activates the box, the second click is inserted check
http://i.xomf.com/hbdxh.png

**Comment by Pringlez** [comment]
I've notice this myself, I look into it when I get the chance

**Comment by Cele91** [comment]
Goods

**Comment by jo3stevens** [comment]
Hey @Cele91 

Again I've not come across this. Which browser are you using?

Cheers

**Comment by jo3stevens** [comment]
Ok now I see it with the new design changes.

**Comment by jo3stevens** [comment]
@Pringlez , dude did you not test any of this? I'm kinda regretting accepting your pull request now. I just dug into the checkboxes and they don't work at all. You're not initiating them with the jQuery plugin to start with, and even doing that it happens before angular has done it's binding so they all appear false. I was able to create a quick directive to watch the model value and update the check boxes so the initial value was correct, but because the plugin is modifying the underlying checkbox via JS it's not updating the Angular model meaning any checkbox changes aren't saved.

There is an angular implementation of the bootstrap switch plugin which should work, otherwise will need to revert them back to regular checkboxes.

**Comment by Pringlez** [comment]
@jo3stevens I tried testing most of functionality but didn't get around to fixing the checkboxes yet, believe me there were other issues like reading the configuration files, authentication, saving config to file etc that wasn't working at all until myself & @surtic86 came along...

The bootstrap plugin switch should do the trick.

**Comment by jo3stevens** [comment]
I don't doubt that @Pringlez; I hadn't updated the project in over and year and were lots of updates to the server config files in that time. I certainly appreciate the contributions, but would be good to ensure things are properly tested before making a pull request.

Cheers

**Comment by jo3stevens** [comment]
The version of bootstrap switch added by @Pringlez was either old or modified. I assume it came with the bootstrap dashboard? To get it working I'd have needed to create quite a complex directive, so I just opted to replace it with the latest version of bootstrap switch and the angular wrapper already written for it.

I've just pushed the changes. It's replaced the checkboxes with yes/no switches. The binding all seems find yet there is still an intermittent error where not all values on a page are being saved which I'll need to look into.
